pkg/sdk/row_access_policies_gen_test.go

```diff
@@ -0,0 +1,245 @@
+package sdk
+
+import "testing"
+
+func TestRowAccessPolicies_Create(t *testing.T) {
+	id := RandomSchemaObjectIdentifier()
+
+	// Minimal valid CreateRowAccessPolicyOptions
+	defaultOpts := func() *CreateRowAccessPolicyOptions {
+		return &CreateRowAccessPolicyOptions{
+			name: id,
+			args: []CreateRowAccessPolicyArgs{{
+				Name: "n",
+				Type: DataTypeVARCHAR,
+			}},
+			body: "true",
+		}
+	}
+
+	t.Run("validation: nil options", func(t *testing.T) {
+		var opts *CreateRowAccessPolicyOptions = nil
+		assertOptsInvalidJoinedErrors(t, opts, ErrNilOptions)
+	})
+
+	t.Run("validation: valid identifier for [opts.name]", func(t *testing.T) {
+		opts := defaultOpts()
+		opts.name = NewSchemaObjectIdentifier("", "", "")
+		assertOptsInvalidJoinedErrors(t, opts, ErrInvalidObjectIdentifier)
+	})
+
+	t.Run("validation: conflicting fields for [opts.OrReplace opts.IfNotExists]", func(t *testing.T) {
+		opts := defaultOpts()
+		opts.OrReplace = Bool(true)
+		opts.IfNotExists = Bool(true)
+		assertOptsInvalidJoinedErrors(t, opts, errOneOf("CreateRowAccessPolicyOptions", "OrReplace", "IfNotExists"))
+	})
+
+	t.Run("validation: [opts.args] should be set", func(t *testing.T) {
+		opts := defaultOpts()
+		opts.args = []CreateRowAccessPolicyArgs{}
+		assertOptsInvalidJoinedErrors(t, opts, errNotSet("CreateRowAccessPolicyOptions", "args"))
+	})
+
+	t.Run("validation: [opts.body] should be set", func(t *testing.T) {
+		opts := defaultOpts()
+		opts.body = ""
+		assertOptsInvalidJoinedErrors(t, opts, errNotSet("CreateRowAccessPolicyOptions", "body"))
+	})
+
+	t.Run("one parameter", func(t *testing.T) {
+		opts := defaultOpts()
+		assertOptsValidAndSQLEquals(t, opts, "CREATE ROW ACCESS POLICY %s AS (n VARCHAR) RETURNS BOOLEAN -> true", id.FullyQualifiedName())
+	})
+
+	t.Run("two parameters", func(t *testing.T) {
+		opts := defaultOpts()
+		opts.args = []CreateRowAccessPolicyArgs{{
+			Name: "n",
+			Type: DataTypeVARCHAR,
+		}, {
+			Name: "h",
+			Type: DataTypeVARCHAR,
+		}}
+		assertOptsValidAndSQLEquals(t, opts, "CREATE ROW ACCESS POLICY %s AS (n VARCHAR, h VARCHAR) RETURNS BOOLEAN -> true", id.FullyQualifiedName())
```

Comment:
is it possible to create a row access policy with no arguments?

Reply:
Not according to the docs.

There is unit test for no args:
```
	t.Run("validation: [opts.args] should be set", func(t *testing.T) {
		opts := defaultOpts()
		opts.args = []CreateRowAccessPolicyArgs{}
		assertOptsInvalidJoinedErrors(t, opts, errNotSet("CreateRowAccessPolicyOptions", "args"))
	})
```
so we do not allow it.